package.json

```diff
@@ -24,7 +24,7 @@
     "trailingComma": "none"
   },
   "peerDependencies": {
-    "eslint-plugin-typescript": "*",
-    "typescript-eslint-parser": "*"
+    "@typescript-eslint/eslint-plugin": "^1.6.0",
+    "@typescript-eslint/parser": "^1.6.0"
```

Comment:
And won't this require a change in `index.js`?

Reply:
You are totally right, I have done as suggested in https://www.npmjs.com/package/@typescript-eslint/parser#usage